proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -120,7 +120,7 @@ message WorkflowExecutionInfo {
     string first_execution_run_id = 55;
     ExecutionStats execution_stats = 56;
     google.protobuf.Timestamp workflow_run_expiration_time = 57 [(gogoproto.stdtime) = true];
-    int64 last_history_node_txn_id = 58;
+    int64 last_first_event_txn_id = 58;
```

Comment:
Can you add comment here? Something like:
```suggestion
    // Transaction Id of the first event in the last batch of events.
    int64 last_first_event_txn_id = 58;
```
and other places in proto where we have `first_last`.